code/game/objects/items/weapons/gift_wrappaper.dm

```diff
@@ -317,14 +317,26 @@
 ///Loot crate///
 /obj/item/weapon/winter_gift/dorkcube
 	name = "loot box"
-	desc = "Don't forget to shout the magic phrase when you open it! It has a tag on it: Made by GIBLOOTPLS."
+	desc = "Don't forget to shout the magic phrase when you open it! It smells faintly of iron."
 	icon = 'icons/obj/storage/storage.dmi'
 	icon_state = "lootbox_purple"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/toolbox_ihl.dmi', "right_hand" = 'icons/mob/in-hand/right/toolbox_ihr.dmi')
 	item_state = "lootbox_purple"
 
 /obj/item/weapon/winter_gift/dorkcube/attack_self(mob/user)
-	user.say("Loot get!")
-	playsound(src, 'sound/misc/achievement.ogg', 30, 1)
-	user.gib()
-	qdel(src)
+	to_chat(user, "<span class='notice'>You begin ripping into the wrapper.</span>")
+	playsound(src, 'sound/items/poster_ripped.ogg', 100, 1)
+	if (do_after(user, src, 2 SECONDS))
+		var/turf/T = get_turf(user)
+		var/mob/living/carbon/human/blooddonor = new /mob/living/carbon/human
```

Comment:
it's actually completely fine to call add_blood without a human, it just won't copy over dna/viruses which we don't care about anyway -- instead of creating a whole human with tons of vars just to gib it, why not just call `add_blood()`?

Reply:
No DNA means that there's no blood overlays on whoever gets bloodied.
Rather than spend an hour figuring out why that was made that way, I figured just generating some random human to steal his blood would be easier. It's a lot more expensive, but I don't expect these to get used that often anyway.

Reply:
That does not appear to be true. Tested on live.

Reply:
![image](https://user-images.githubusercontent.com/8468269/64107460-ded95500-cd7a-11e9-8c0b-e3b80494434d.png)
I called `add_blood` on both of these humans, and both their uniforms.
The left one was without arguments, the right with a nearby human as argument.
This was tested on live. Try it. Do you see my point?

Reply:
It works for items, something must be different for mobs

Reply:
Yeah, it is, that's why I did it like this. If you think I should invest my time into figuring out why things are this way, and if I can just cut that bit out, let me know.

Reply:
Looked into this more, the dummy is fine